developers/weaviate/starter-guides/custom-vectors.mdx

```diff
@@ -8,187 +8,207 @@ image: og/docs/tutorials.jpg
 
-Later, when you work with the stored vectors, Weaviate uses a vectorized version of your query to search the vector space.
+Weaviate has [integrations](/developers/weaviate/model-providers) with [vectorizer](../modules/retriever-vectorizer-modules/index.md) models so you can create vector embeddings at import time. An alternative approach is to "bring your own vectors," and supply the vector embeddings when you upload your data. The embeddings could be custom vectors or pre-existing vectors from one of the popular model providers. This guide discusses importing data that already has vectors.
 
-This guide discusses importing data that already has vectors. An alternative approach is to use a [vectorizer](../modules/retriever-vectorizer-modules/index.md) model that generates vectors at import time. Queries have to be vectorized using the same vectorizer as the data objects. When you define your collection schema, be sure to define the same vectorizer that you use to vectorize the data.
+To import data with your own vectors, follow these steps:
 
-Weaviate supports embeddings generated by popular third party services and custom vector embeddings that you create. For details on how to connect your client application to a vectorizer, see the follow pages:
+1. [Connect to a Weaviate instance.](#connect-to-weaviate)
+1. [Import data that specifies its own vectors.](#import-data-and-vectors)
+1. [Perform a vector search.](#query)
 
-- [Supported models](/developers/weaviate/modules/retriever-vectorizer-modules)
-- [Custom models](/developers/weaviate/modules/other-modules/custom-modules)
+## Setup
 
-## Guide outline
+This section introduces setup requirements and the data that this example uses.
 
-This guide uses a free sandbox in [Weaviate Cloud](/developers/wcs/sandbox). If you don't have a Weaviate Cloud account, you can [create one](/developers/wcs/create-account) or else follow along on a Weaviate instance of your own.
+### Example Data
 
-Follow these steps to import data that includes vectors.
+The example dataset is 10 questions from the popular quiz show "Jeopardy!". Each of the questions has an associated vector embedding.
 
-1. [Create a sandbox instance](#create-a-sandbox-instance)
-1. [Connect to your sandbox](#connect-to-your-sandbox)
-1. [Prepare the collection](#prepare-the-collection)
-1. [Import the data](#import-the-data)
+In this example, the vector embeddings were obtained from OpenAI API, by using the [`text-embedding-ada-002` model](https://platform.openai.com/docs/guides/embeddings/what-are-embeddings). Your vectors can come from any source, perhaps from your own vectorizer model or a different service provider such as Cohere or Hugging Face.
```

Comment:
I propose to put this into the `sample data` dropdown. The provenance of the vectors are immaterial for this guide in my opinion.